packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -1338,12 +1360,12 @@ describe("Get (shared) versions of attribute", () => {
                 const result1 = await services1.consumption.attributes.getSharedVersionsOfRepositoryAttribute({ attributeId: version.id });
                 expect(result1.isSuccess).toBe(true);
                 const returnedVersions1 = result1.value;
-                expect(returnedVersions1).toStrictEqual([sOSIAVersion2, sOSIAVersion2FurtherPeer]);
+                expect(new Set(returnedVersions1)).toStrictEqual(new Set([sOSIAVersion2, sOSIAVersion2FurtherPeer]));
```

Comment:
out of curiosity - why do we need that?

Reply:
The order in which the (databank) query returns copies of the same version shared with different peers varies. Using Set, the order of the compared arrays is neglected.

Reply:
IMO these results should be sorted so this should not be nessecary